src/Tizen.NUI/src/internal/Interop/Interop.WebView.cs

```diff
@@ -300,6 +300,9 @@ internal static partial class WebView
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_WebView_RegisterContextMenuHiddenCallback")]
             public static extern void RegisterContextMenuHiddenCallback(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_WebView_GetPlainTextAsynchronously")]
+            public static extern void GetPlainTextAsynchronously(global::System.Runtime.InteropServices.HandleRef jarg1, global::System.Runtime.InteropServices.HandleRef jarg2);
```

Comment:
Could we use more understandable naming here?
jarg1, jarg2 is difficult to understand, even though it is well used manner until now.

Reply:
Thank you for the review. I renamed parameters!